server/models/bill.coffee

```diff
@@ -2,17 +2,22 @@ cozydb = require 'cozydb'
 
 
 module.exports = Bill = cozydb.getModel 'Bill',
```

Comment:
I tried to comment what each field should contain. Not sure I guessed all of them correctly though.